test/test.py

```diff
@@ -45,9 +58,17 @@ def test(
         model = model_dir.parent.name
 
         if split == 'test':
-            eval_paths = [site_data_dir / 'test.csv']
-            # TODO: Run ensemble model
-        
+            eval_df = pd.read_csv(site_data_dir / 'test.csv')
+            model_obj = EnsembleModel(model_dir)
+            scores = {}
+            for eval_metric in eval_metrics:
```

Comment:
**Problem**
1. Only need one prediction for all metric evaluations. Now each evaluation runs one prediction. 
2. y and y_hat are better to be saved in a csv for double checking.

**Fix**
Make predictions of testing data before looping metrics and save a copy

Reply:
Great points - done!